salt/utils/ansible.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import os
+
+import salt.modules.cmdmod
+import salt.utils.json
+import salt.utils.path
+import salt.utils.stringutils
+from salt.exceptions import CommandExecutionError
+
+__virtualname__ = "ansible"
+
+log = logging.getLogger(__name__)
+
+
+def __virtual__():  # pylint: disable=expected-2-blank-lines-found-0
+    if salt.utils.path.which("ansible-inventory"):
+        return __virtualname__
+    return (False, "Install `ansible` to use inventory")
+
+
+def targets(inventory="/etc/ansible/hosts", **kwargs):
```

Comment:
Again, why `**kwargs` and not explicit keyword arguments?

Reply:
Fixed.